lib/widgets/product/product_page.dart

```diff
@@ -3,26 +3,12 @@ import 'package:google_mobile_ads/google_mobile_ads.dart';
 import 'package:provider/provider.dart';
 import 'package:retro_shopping/helpers/ad_state.dart';
 import 'package:retro_shopping/helpers/constants.dart';
-
+import 'package:retro_shopping/model/product_model.dart';
 import '../../helpers/app_icons.dart';
 import '../retro_button.dart';
 
 // ignore: must_be_immutable
 class ProductPage extends StatefulWidget {
-  ProductPage(
```

Comment:
You can have the Product Object here, which will get the object from the route_page.dart